Cargo.toml

```diff
@@ -21,8 +21,9 @@ exclude = [".github"]
 [dependencies]
 clap = { version = "4.0.8", features = ["derive"] }
 threadpool = "1.8.1"
-termcolor = "1.0.5"
 escape8259 = "0.5.2"
+anstream = "0.6.14"
```

Comment:
Note: this achieves de-duplication for everyone using clap v4.4.6 or later. Earlier clap versions (for MSRV) work, they might just pull in two versions of anstream. That's alright for me, but raises the question of what happens if and when anstream has another semver-breaking release and clap adopts that. This is purely hypothetical at this point, but I think I'd prefer libtest-mimic to follow latest clap as much as feasible. Alternatively, if the breaking changes are small enough, it could support a range of anstream versions, just like some crates depend on `itertools = ">= 0.10, <= 0.12"` or similar.